### ISSUE TITLE
Multiple progress

### ISSUE THREAD

**Issue by Kangcor** [vmware/clarity#3406]
## Summary

It would be nice to have an horizontal stacked progress bar to show summarized data

* What is the change? Add support to multiple progress elements inside progress blocks
* Why should it go in Clarity? It is a common use case and supported in other frameworks like [bootstrap](https://getbootstrap.com/docs/4.3/components/progress/#multiple-bars)
* Does this change impact existing behaviors? If so how? Only adds a new use for progress, doesn't break existing code
* If this change introduces a new behavior, is this behavior accessible? I think the same accessibility is maintained with multiple progress, like vertical stacked progress groups.

## Use case

With horizontal progress bar it's possible to see the same data like vertical progress groups, with an addition to see the total % that adds up with all the progress bars together. A use case can be to represent different states of something changing in time, like a timeline with active and inactive states of something that changes over time.

## Examples

[bootstrap](https://getbootstrap.com/docs/4.3/components/progress/#multiple-bars)
[conciseCss](https://concisecss.com/archive/v2.0.0/add-ons/progress-bars/)
[ClayUI](https://clayui.com/docs/css-framework/satellites/progress-bars.html)
## Workarounds

I didn't have any at the moment

**Comment by gnomeontherun** [comment]
We have stacked progress bars, which I believe would be a workaround. Otherwise, you could approximate this by positioning several bars on top of one another with CSS in the meantime. 

I don't know if this is a good pattern or not, because then we rely solely on color to differentiate and that can be problematic. I'm going to mark this as needs UX input to see if this is wise to recommend or not.

**Comment by pumano** [comment]
Looks like I need that feature too, but in my mind stacked horizontal progress bar should be supported by`card with progress bar` component too :) Also current vertical stacked progress bars not supported by card (i don't try it, but documentation don't have data about it).

**Comment by colinreedmiller** [comment]
We need a more detailed description of use cases. Clarity progress bars support a number of needs and contexts - in order to determine if we should add this enhancement and if so, what attributes it will support, we need more specifics.

How would horizontally stacked work in the context of a card?
What kind of legend would be needed to understand the content?
Is additional specific information (i.e. label and %) needed?

Stacked layout (already in Clarity) conveys information more clearly than horizontally stacked (several progress bars sharing the same progress container) and this is recommended. If the intent in this request is for information to be conveyed purely with color this is an a11y concern. There is a compact version where space is an issue.

**Comment by bmireles** [comment]
We designed a component like this for a recent feature in vSphere. We used it to provide the user with a visualization of how many resource were within a particular range. We didn't solely rely on color due to accessibility concerns. We wanted the color to play a supporting role as a visual queue. The main information is relayed in the data to the left and right of the bars.

<img width="676" alt="Screen Shot 2019-07-19 at 5 31 45 PM" src="https://user-images.githubusercontent.com/18449355/61569100-5f522980-aa4b-11e9-8639-733c5f553ee2.png">

**Comment by colinreedmiller** [comment]
There are a11y concerns with color as informative coding in progress bars that requires a design update to be complaint. see: #3517 

Making an enhancement to multiple progress to include horizontally stacked will not be undertaken in the near term. Marking this low priority and adding a11y flag. Stacked progress as exist in Clarity will need to suffice for the present.

**Comment by mathisscott** [comment]
Closing for a11y issues. This is not a pattern we can adopt.
